lib/react_phoenix/react_io.ex

```diff
@@ -1,5 +1,7 @@
 defmodule ReactPhoenix.ReactIo do
   @moduledoc false
 
-  use StdJsonIo, otp_app: :react_phoenix, script: Application.get_env(:react_phoenix, :react_stdio_path, ReactPhoenix.ReactIo.PathFinder.react_stdio_path())
+  use StdJsonIo,
+    otp_app: :react_phoenix,
+    script: (Application.get_env(:react_phoenix, :react_stdio_path) || ReactPhoenix.ReactIo.PathFinder.react_stdio_path())
```

Comment:
I can revert that change if you'd like, but I switched to the `||` deliberately.  The `||` short-circuits the operation, so that if the env variable is set, then `ReactPhoenix.ReactIo.PathFinder.react_stdio_path` is never called.  

That means that developers who set the env variable won't see the warning, because we never search the default locations.